production/technical_userguide.md

```diff
@@ -236,25 +238,44 @@ Metadata about the Beneficiary Claims Data API is available as a FHIR [Capabilit
-* [**Patient**](https://www.hl7.org/fhir/patient.html){:target="_blank"}: the information in this Resource Type can be thought of as your CCLF files 8 and 9: this is where you get your information about who your beneficiaries are, their demographic information, and updates to their patient identifiers.
-* [**Coverage**](https://www.hl7.org/fhir/coverage.html){:target="_blank"}: the information in this Resource Type indicates the beneficiary’s enrollment record. It includes information on a beneficiary’s Part A, Part B, Part C, and Part D coverage, as well as markers for End-stage Renal Disease (ESRD) and Old Age and Survivors Insurance (OASI).
+#### The FHIR Resource Types Offered through the API
+* **[ExplanationOfBenefit](https://www.hl7.org/fhir/explanationofbenefit.html){:target="_blank"}:** this Resource Type provides the same information ACOs receive in CCLF files 1-7. This file contains the lines within an episode of care, including where and when the service was performed, the diagnosis codes, the provider who performed the service, and the cost of care.
+* **[Patient](https://www.hl7.org/fhir/patient.html){:target="_blank"}:** the information in this Resource Type can be thought of as an ACO’s CCLF files 8 and 9: this is where you get information about who your beneficiaries are, their demographic information, and updates to their patient identifiers.
+* **[Coverage](https://www.hl7.org/fhir/coverage.html){:target="_blank"}:** the information in this Resource Type indicates the beneficiary’s enrollment record. It includes information on a beneficiary’s Part A, Part B, Part C, and Part D coverage, as well as markers for End-stage Renal Disease (ESRD) and Old Age and Survivors Insurance (OASI).
 
-You can make a request to the `Patient` endpoint for all three Resource Types at once, one at a time, or a combination of any two together. This section describes a request for all three Resource Types; information about requesting one at a time follows below.
+#### The BCDA/bulk FHIR Endpoints
+- **Patient Endpoint:** Request `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter.
+- **Group Endpoint:** Request `ExplanationOfBenefit`, `Patient`, and/or `Coverage` data for all beneficiaries. Filter data by date using the `_since` parameter.
+  - **Note: If you filter data using _since within the /Group endpoint, you will receive claims data since the date of your choice for existing beneficiaries AND you will also receive 7 years of historical data for all beneficiaries that are newly attributed to your ACO. Learn more about how to retrieve the most current beneficiary claims data for existing beneficiaries AND historical data for newly attributed beneficiaries.** 
```

Comment:
(1) needs backtick around `_since`, `/Group`

(2) proposed reword: 
> you will receive claims data both since the date of your choice for existing beneficiaries AND 7 years of historical data for all beneficiaries that are newly attributed to your ACO.

Reply:
Fixed backticks